bigquery/deploy/k8s_es_deploy/deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Move script up a dir, since eventually it will have indexer commands?

Reply:
The vast majority of the time I want to run only one or the other, so I'd like to have separate scripts for es setup + indexer, with maybe an overarching script that runs both.